ow_lander/src/ow_actions/common.py

```diff
@@ -0,0 +1,46 @@
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+from math import pi, tau
+from ow_actions import constants
+# import roslib; roslib.load_manifest('urdfdom_py')
+from urdf_parser_py.urdf import URDF
+
+class Singleton(type):
+  _instances = {}
+  def __call__(cls, *args, **kwargs):
+    if cls not in cls._instances:
+      cls._instances[cls] = super(Singleton, cls).__call__(*args, **kwargs)
+    return cls._instances[cls]
+
+def is_shou_yaw_goal_in_range(joint_goal):
```

Comment:
This function seems rather specific, compared to the rest of the file.   Why not make it applicable to any joint?

Reply:
This is left over from a previous implementation (`utils.py`). It may be overly specific, but I would prefer not change more than I have to for this refactor.